### DISCUSSION TITLE
How to convert a PDF from letter to A4 size?

### DISCUSSION THREAD

**Discussion by eabase** [py-pdf/pypdf#2590] [Q&A]
The examples for page size does not include an example for how to convert an already existing document to use another page size. The examples only show how to create a new page, which is not what I want.

**Comment by stefan6419846** [comment]
There is nothing like a default conversion for page sizes as this might depend on your use case. The easiest approaches are to use `page.scale_to` or just update the corresponding page boxes (see the docs at https://pypdf.readthedocs.io/en/stable/modules/PageObject.html) to the desired size.
